### ISSUE TITLE
508-defect-3 [SCREENREADERS]: Word "veteran" in search suggestions mispronounced and search suggestions are spoken twice

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#24784]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Point of Contact: Angela Fowler

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect my screen reader to pronounce common words correctly and avoid unnecessary repetition.
<h2>Details</h2>
In Firefox, using JAWS or NVDA, when I arrow through a list of typeahead suggestions, my screen reader says each suggestion twice. The first time it mispronounces the word "veteran" as though it were misspelled. The second time it pronounces it correctly. This issue does not occur in Chrome.

<h2>Acceptance Criteria</h2>
- [ ] JAWS and NVDA read search suggestions once and pronounce them correctly in Firefox.
<h2>Environment</h2>
<ul>
<li>JAWS and Firefox</li>
<li>NVDA and Firefox</li>
</ul>
<h2>Steps to Recreate</h2>
<ul>
<li>Using Firefox with either JAWS or NVDA, Access staging.va.gov </li>
<li>In the search box, type the word "vet" to generate a list of search results containing the word "veteran."</li>
<li>Arrow through the list, noting how the screen reader responds</li>
</ul>

**Comment by denisecoveyduc** [comment]
Hi @joshkimux @AngelaFowler82 - @srsuddath did some exploration and as per his [comment here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27307#issuecomment-886999994) we are unable to resolve this at this time. 

We are however preparing to create a typeahead drop down component which will give us more control. We can circle back on this topic then.
